src/move/econia/sources/registry.move

```diff
@@ -514,6 +519,62 @@ module econia::registry {
         MarketCounts{n_markets, n_recognized_markets}
     }
 
+    #[view]
+    /// Return optional market ID corresponding to given market
+    /// parameters when the base asset is a coin type.
+    ///
+    /// Restricted to private view function to prevent runtime
+    /// transaction collisions against the registry.
+    ///
+    /// # Testing
+    ///
+    /// * `test_set_remove_check_recognized_markets()`
+    fun get_market_id_base_coin<BaseType, QuoteType>(
+        lot_size: u64,
+        tick_size: u64,
+        min_size: u64,
+    ): Option<u64>
+    acquires Registry {
+        get_market_id(MarketInfo{
+            base_type: type_info::type_of<BaseType>(),
+            base_name_generic: string::utf8(b""),
```

Comment:
Can you help me understand what this string-represented value means?
Is it something like `0xbeef::test_eth::TestETH` where this value is just the string representation of a generic type, or something else?

Reply:
From @alnoki:

> The base asset for a market doesn’t have to be a coin
https://econia.dev/overview/registry#underwriters